### ISSUE TITLE
Problem with fa-book

### ISSUE THREAD

**Issue by wgmv** [FortAwesome/Font-Awesome#12662]
I have a problem with the fa-book icon using the light version 5.0.8 (not sure about others)

The icon is added in the same way as other icons but appears to be smaller (17.5px instead of 20px). The generated code shows

`<svg clsas="svg-inline--fa fa-book fa-w-14`

while the other icons use fa-w-16 instead of fa-w-14

How can I fix this or is it a bug? I found another issue with this icon https://github.com/FortAwesome/Font-Awesome/issues/5185 but not sure if related.

See attached image for a demonstration (as you can see the Text is closer to the icon)

![fa-book](https://user-images.githubusercontent.com/6874094/37713116-2bccf74c-2d16-11e8-9c90-ebbf1f0c089a.jpg)

**Comment by acshef** [comment]
@wgmv see [Fixed-Width Icons](https://fontawesome.com/how-to-use/svg-with-js#fixed-width-icons)

tl;dr use the class `fa-fw`

**Comment by wgmv** [comment]
thanks this solves my issue.
